ethers-solc/src/compile/output/mod.rs

```diff
@@ -278,7 +278,9 @@ impl<T: ArtifactOutput> ProjectCompileOutput<T> {
     /// # }
     /// ```
     pub fn find(&self, path: impl AsRef<str>, contract: impl AsRef<str>) -> Option<&T::Artifact> {
-        let contract_path = path.as_ref();
+        let mut current_dir = env::current_dir().unwrap();
+        current_dir.push(path.as_ref());
+        let contract_path = current_dir.to_str().unwrap();
```

Comment:
I'm strongly against this here.
There's no guarantee that the cwd is the root dir.

what's an example where you'd need this?

Reply:
I'm just following a tutorial to deploy a demo contract, but failed with getting the contract from the output after compiling. Is just annoying to setting the absolute path of the contract.

Okay, i got you point. So, it's proper to use `find_first` method?

Reply:
> Is just annoying to setting the absolute path of the contract.

right, there are functions to strip/join the root path for example.

`find_first` returns the first matching contract by name, if this is unique this is easier to use.